doc/releases/release-notes-3.6.rst

```diff
@@ -261,6 +263,29 @@ Drivers and Sensors
   * The Legacy Virtual Register defines have been renamed from ``I3C_DCR_I2C_*``
     to ``I3C_LVR_I2C_*``.
 
+  * Added the ability to specify a start address when searching for a free I3C
+    address to be reserved. This requires a new function argument to
+    :c:func:`i3c_addr_slots_next_free_find`.
+
+  * ``i3c_cdns``:
+
+    * Added support to handle controller abort where target does not emit
+      end of data for register read but continues sending data.
+
+    * Updated the timeout calculation to be coupled with CPU speed instead of
+      a fixed number of retries.
+
+  * ``i3c_mcux``:
+
+    * Fixed ``mcux_i3c_config_get()`` of not returning the configuration to caller.
+
+    * Sped up the FIFO read routine to support higher transfer rate.
```

Comment:
nit: spelling of `sped` should be `speed`

Reply:
It is supposed to be the past tense of speed... at least Google said so.

Reply:
for the record, it threw me off as well when I first reviewed this PR, and add to read it 3x times to get it :) My guess is that there is a variation of this sentence starting with "Improved ..." that would work just as well and would suit better the "added / updated / fixed" framework we tend to adopt for the release notes
</nit>